src/test/java/org/folio/rest/impl/RtacResourceImplTest.java

```diff
@@ -1,545 +1,294 @@
-
-    // Test done
-    logger.info("Test done");
+  void shouldRespondWithInternalServerError_whenErrorOccurredWhileRetrievingLoans(
+      VertxTestContext testContext) {
+    testContext.verify(
+        () -> {
+          String validInstanceIdsJson =
+              pojoToJson(MockData.RTAC_REQUEST_WITH_INSTANCE_ID_LOAN_STORAGE_ERROR);
+          RequestSpecification request = createBaseRequest(validInstanceIdsJson);
+          request
+              .when()
+              .post()
+              .then()
+              .statusCode(500)
+              .contentType(ContentType.TEXT)
+              .body(is(SERVER_ERROR));
+          testContext.completeNow();
+        });
   }
 
   @Test
-  public final void testGetRtacByIdNoHoldings() {
-    logger.info("Testing for successful RTAC by instance id with no holdings");
-
-    final Response r = RestAssured
-        .given()
-          .header(tenantHeader)
-          .header(urlHeader)
-          .header(contentTypeHeader)
-        .when()
-          .get("/rtac/1613697d-5e18-49b0-9613-08443b87cbc7")
-        .then()
-          .contentType(ContentType.JSON)
-          .statusCode(200)
-          .extract()
-          .response();
-
-    final String body = r.getBody().asString();
-    final JsonObject json = new JsonObject(body);
-
-    assertEquals(0, json.getJsonArray("holdings").size());
-
-    // Test done
-    logger.info("Test done");
+  void shouldRespondWithInternalServerError_whenErrorOccurredWhileRetrievingInstances(
+      VertxTestContext testContext) {
+    testContext.verify(
+        () -> {
+          String validInstanceIdsJson =
+              pojoToJson(MockData.RTAC_REQUEST_WITH_INSTANCE_ID_LOAN_STORAGE_ERROR);
+          RequestSpecification request = createBaseRequest(validInstanceIdsJson);
+          request
+              .when()
+              .post()
+              .then()
+              .statusCode(500)
+              .contentType(ContentType.TEXT)
+              .body(is(SERVER_ERROR));
+          testContext.completeNow();
+        });
   }
 
   @Test
-  public final void testGetRtacByIdNoItems() {
-    logger.info("Testing for successful RTAC by instance id with no items");
-
-    final Response r = RestAssured
-        .given()
-          .header(tenantHeader)
-          .header(urlHeader)
-          .header(contentTypeHeader)
-        .when()
-          .get("/rtac/2ae0635e-5534-4b7d-b28f-f0816329baa3")
-        .then()
-          .contentType(ContentType.JSON)
-          .statusCode(200)
-          .extract()
-          .response();
+  void shouldReturnEmptyHoldings_whenInstanceHasNotItems(VertxTestContext testContext) {
+    testContext.verify(
+        () -> {
+          String validInstanceIdsJson =
+              pojoToJson(MockData.RTAC_REQUEST_WITH_INSTANCE_NO_ITEMS_AND_HOLDINGS);
+          RequestSpecification request = createBaseRequest(validInstanceIdsJson);
+          String body =
+              request
+                  .when()
```

Comment:
Do you find the wall of text method calls more readable than the given/when/then blocks that were previously there?

Reply:
I think so. This has been formatted with google formatter.